src/main/java/net/minecraftforge/common/property/Properties.java

```diff
@@ -6,9 +6,11 @@
 
 import org.apache.commons.lang3.reflect.ConstructorUtils;
 
+import com.google.common.base.Optional;
+
 public class Properties
 {
-    public static <P extends IProperty, V>IUnlistedProperty<V> toUnlisted(P property)
+    public static <P extends IProperty, V>IUnlistedProperty toUnlisted(P property)
```

Comment:
If I remember correctly, this should be like it was previously... I will double check.